code/modules/mob/living/basic/jungle/seedling/seedling.dm

```diff
@@ -0,0 +1,314 @@
+#define SEEDLING_STATE_NEUTRAL 0
+#define SEEDLING_STATE_WARMUP 1
+#define SEEDLING_STATE_ACTIVE 2
+
+//A plant rooted in the ground that forfeits its melee attack in favor of ranged barrages.
+//It will fire flurries of solar energy, and occasionally charge up a powerful blast that makes it vulnerable to attack.
+/mob/living/basic/seedling
+	name = "seedling"
+	desc = "This oversized, predatory flower conceals what can only be described as an organic energy cannon."
+	icon = 'icons/mob/simple/jungle/seedling.dmi'
+	icon_state = "seedling"
+	icon_living = "seedling"
+	icon_dead = "seedling_dead"
+	habitable_atmos = list("min_oxy" = 2, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minimum_survivable_temperature = 0
+	maximum_survivable_temperature = 450
+	mob_biotypes = MOB_ORGANIC | MOB_PLANT
+	maxHealth = 100
+	health = 100
+	pixel_y = -14
+	response_harm_continuous = "strikes"
+	response_harm_simple = "strike"
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	lighting_cutoff_green = 20
+	lighting_cutoff_blue = 25
+	mob_size = MOB_SIZE_LARGE
+	ai_controller = /datum/ai_controller/basic_controller/seedling
+	///the state of combat we are in
+	var/combatant_state = SEEDLING_STATE_NEUTRAL
+	///the colors our petals can have
+	var/list/possible_colors = list(COLOR_RED, COLOR_YELLOW, COLOR_OLIVE, COLOR_CYAN)
+	///appearance when we are in our normal state
+	var/static/mutable_appearance/petal_neutral
+	///appearance when we are in our warmup state
+	var/static/mutable_appearance/petal_warmup
+	///appearance when we are in the firing state
+	var/static/mutable_appearance/petal_active
+	///appearance when we are dead
+	var/static/mutable_appearance/petal_dead
+	///the bucket we carry
+	var/obj/item/reagent_containers/cup/held_can
```

Comment:
You should implement `/mob/living/basic/seedling/Exited(...)` to handle the reference to this